### ISSUE TITLE
Mouse wheel speed control

### ISSUE THREAD

**Issue by sherlockmac** [baianat/hooper#179]
**Is your feature request related to a problem? Please describe.**
The carousel jumps to the next slide pretty fast when using a MacBook Pro trackpad.

**Describe the solution you'd like**
It would be great if there'd be a prop to control the threshold of scroll distance or time.

**Additional context**
I think the prop should be implemented in the onWheel function.

**Comment by alexborecky** [comment]
I have the same problem. Also, when you scroll through the whole carousel, it starts glitching...

**Comment by saltymouse** [comment]
Duplicate of #170

**Comment by badoubadou** [comment]
Same problem. Someone found a solution ? This is not production ready, at least for Nuxt... It should be said somewhere.

**Comment by MGuadalupe** [comment]
Same problem here, any solution at least for Nuxt.js?

**Comment by badoubadou** [comment]
no, just dumped hooper, with nuxt it's not the only issues it has. You can't use nuxt-link in it too.
